content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -28,23 +28,23 @@ In **FY22Q4**, we conducted discovery work to better understand the bottlenecks
 
 #### Details
 
-**Monorepo performance**: is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, and identified the unindexed monorepo performance is still poor, and several facets of search on large monorepos cause significant load on gitserver.
+**Monorepo performance**: is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver.
 
-**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results, using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements, and drive the areas of ranking we choose to focus on.
+**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements and drive the areas of ranking we choose to focus on.
 
 **Code host coverage**: as we seek to keep expanding the code we index to include more non-GitHub.com or GitLab.com code hosts, we will add support for package host integrations (PyPi, Rubygems, NuGet, Crates, proxy.golang.org). This will not only increase our code host coverage, but also be a stepping stone towards unblocking [use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#).
 
 #### What we're not working on and why
 
 ##### Scaling sourcegraph.com to 5.5M repositories
```

Comment:
@eugeniaft I mentioned in the last update that this section is pretty long - it made sense to be when the decision was new, but now that it has been made I think it might be able to be summarized down to a paragraph or two for the purposes of this page (i.e., summarize the decision instead of present the reasoning in detail). Up to you in the end, and can definitely be done via a separate PR, but wanted to mention.

Reply:
Yes, good point. I'll work on that in a separate PR. Thanks!